code/procs/string_picker.dm

```diff
@@ -34,11 +34,11 @@ var/global/list/smart_string_pickers = list()
 
 /datum/smart_string_picker
 	//var/static/regex/head_splitter = new(@"\s*([\l_\d]+)\s*:=\s*", "igm")
-	var/static/regex/section_splitter = new(@(##)(?:"?\s*\n\s*\n+\s*"?|"\s*$)##, "m")
-	var/static/regex/sentence_splitter = new(@(##)"?\s*(?:@,\n?|\n)\s*"?##, "m")
-	var/static/regex/bracket_splitter = new(@(##)\[\s*|\s*\)?\s*\]##)
-	var/static/regex/in_brackets_crap = new(@(##)\s*[(,:]\s*##)
-	
+	var/static/regex/section_splitter = new({"(?:"?\\s*\\n\\s*\\n+\\s*"?|"\\s*$)"}, "m")
```

Comment:
I think using `@{"..."}` is more readable than escaping the backslashes. This specifically would be `@{"(?:"?\s*\n\s*\\n+\s*"?|"\s*$)"}`. If the parser can use that it might be the better option (applies to the other changed regexes too).